### PR TITLE
Completed javascript-operators-and-expressions exercise. - Tiffany Brown

### PR DESCRIPTION
**Pull request by tiffanydbrown**
Screenshot 2023-10-23 at 2 54 51 PM

### CODE REVIEW

File: javascript-operators-and-expressions/main.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+const width = 12;
+const height = 14;
+const area = width * height;
+
+console.log('Value of the area:', area);
+console.log('Type of the area:', typeof area);
+
+const bill = 199;
+const payment = 50;
+const change = bill - payment;
```
Comment (shawnkost) [inline javascript-operators-and-expressions/main.js:10]: Your `payment` should be larger than the `bill`, and then your operands on line 10 need to be swapped

File: javascript-operators-and-expressions/main.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+console.log('Value of the isAcidic:', isAcidic);
+console.log('Type of isAcidic:', typeof isAcidic);
+
+const headCount = 10;
+const isSparta = headCount === 300;
+
+console.log('Value of isSparta:', isSparta);
+console.log('Type of isSparta:', typeof isSparta);
+
+const motto = fullName + ' ' + ' is the GOAT';
```
Comment (shawnkost) [inline javascript-operators-and-expressions/main.js:42]: This should be broken up into 2 steps. You should originally initialize `motto` and assign it a value of `fullName`.

Then, using the `+=` operator, you would concatenate the string back to `motto` and reassign

**Review by shawnkost — CHANGES_REQUESTED** [review]
Couple changes here Tiffany!

File: javascript-operators-and-expressions/main.js (hunk 1 above)
Comment (tiffanydbrown) [reply javascript-operators-and-expressions/main.js:10]: 👍🏽

Comment (tiffanydbrown) [reply javascript-operators-and-expressions/main.js]: Screenshot 2023-10-23 at 3 14 09 PM

**Comment by tiffanydbrown** [comment]
Screenshot 2023-10-23 at 3 14 09 PM

File: javascript-operators-and-expressions/main.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
```
Comment (shawnkost) [inline javascript-operators-and-expressions/main.js:42]: This still needs to be addressed! 

This should be broken up into 2 steps. You should originally initialize motto and assign it a value of fullName.

Then, using the += operator, you would concatenate the string back to motto and reassign

**Review by shawnkost — CHANGES_REQUESTED** [review]
One more change here that was overlooked!

File: javascript-operators-and-expressions/main.js (hunk 3 above)
Comment (tiffanydbrown) [reply javascript-operators-and-expressions/main.js:42]: Screenshot 2023-10-23 at 4 46 53 PM

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**